<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>状态检测防火墙</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="IPBOX1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>src_ip</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dst_ip</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IPBOX1_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="IPBOX1_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="IPBOX1_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="HS_dstIP1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_dstIP2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_dstIP3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="HS_dstIP4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdstip1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdstip2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdstip3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdstip4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubmit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>上传规则至数据库</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>协议号</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Protocol">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>src_mask</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="hSBsrcmask">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>dst_mask</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditsrcmask">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditdstmask">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskAdd10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>+10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskAdd5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskAdd1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskSub10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskSub5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtondstmaskSub1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>时间范围</string>
    </property>
   </widget>
   <widget class="QComboBox" name="beginning_time_hour">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="beginning_time_minute">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="jiantou">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="endtime_hour">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="endtime_minute">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>默认策略</string>
    </property>
   </widget>
   <widget class="QLabel" name="dice">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ROLL!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditStragy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>规则数据库</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>401</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SubmitAll">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>480</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>确认规则</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="srcport1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="srcport2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="srcport3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="srcport4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="srcport5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dstport2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dstport5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dstport3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dstport4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="dstport1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>42</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号为0则为Any</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Any_Src_ip">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Any Src_ip</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Any_Dst_ip">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Any Dst_ip</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>状态检测防火墙</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
